rest/komand_rest/util/util.py

```diff
@@ -22,9 +27,100 @@ def copy_dict(x):
         return d
 
     @staticmethod
-    def call_api(url, path, headers, ssl_verify, auth=None):
+    def body_object(response) -> dict:
+
+        body_object = {}
         try:
-            response = requests.request("GET", f"{url}{path}", headers=headers, verify=ssl_verify, auth=auth)
+            body_object = response.json()
+        except ValueError:
+            """ Nothing? We don't care if it fails, that could be normal """
+        # It's possible to have a successful call with no body
+        # https://stackoverflow.com/questions/32319845/python-requests-gives-none-response-where-json-data-is-expected
+        if body_object is None:
+            body_object = {}
+
+        return body_object
+
+
+def url_path_join(*parts):
+    """Normalize url parts and join them with a slash."""
+    schemes, netlocs, paths, queries, fragments = zip(*(urlsplit(part.strip()) for part in parts))
+    scheme = first(schemes)
+    netloc = first(netlocs)
+    path = '/'.join(x.strip('/') for x in paths if x)
+    query = first(queries)
+    fragment = first(fragments)
+    return urlunsplit((scheme, netloc, path, query, fragment))
+
+
+def first(sequence, default=''):
+    return next((x for x in sequence if x), default)
+
+
+class RestAPI(object):
+    CUSTOM_SECRET_INPUT = "CUSTOM_SECRET_INPUT"
```

Comment:
```suggestion
    CUSTOM_SECRET_INPUT = "CUSTOM_SECRET_INPUT"  # noqa: B105
```
Ignoring the static analysis check for this - it thinks it's a hardcoded secret :)